src/getHumanAge.test.js

```diff
@@ -7,4 +7,55 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return 0 human years for every 0 years of animals', () => {
+    expect(getHumanAge(0, 0))
+      .toEqual([0, 0]);
+  });
+
+  test('should return 1 human year for first 15 years of animals', () => {
+    expect(getHumanAge(15, 15))
+      .toEqual([1, 1]);
+  });
+
+  test('should return 2 human year for first 24 years of animals', () => {
+    expect(getHumanAge(24, 24))
+      .toEqual([2, 2]);
+  });
+
+  test(`should count third and more human years
+    as 4 for cats and 5 for dogs`, () => {
+    expect(getHumanAge(40, 39))
+      .toEqual([6, 5]);
+  });
+
+  test(`should round down human years
+    if years of animals does not reach an integer value`, () => {
+    expect(getHumanAge(1, 1))
+      .toEqual([0, 0]);
+
+    expect(getHumanAge(14, 14))
```

Comment:
Please, divide all your tests into separate and provide correct summaries for each one

Reply:
what it means??